testing/tests/headless.test.js

```diff
@@ -16,6 +22,16 @@ describe("Basic viewing of functional pages", () => {
     await expect(page).toMatch("<foo>: A test tag");
   });
 
+  it("open the /en-US/docs/Web/InteractiveExample page", async () => {
+    await page.goto(testURL("/en-US/docs/Web/InteractiveExample"), {
+      // Be a bit less patient with this particular page because it contains
+      // an iframe, on an external URL,  which we're not particularly
+      // interested in waiting for.
+      waitUntil: "domcontentloaded",
```

Comment:
I'm actually not sure if this is right! But it seems to work. We basically don't want to depend on waiting for the iframe's src to have fully downloaded everything. 

But in some future we might change our minds about this and raise the timeout (and perhaps do some retries) and check that the contents of the iframe actually works.